tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -21,18 +19,6 @@
 
 PAUSE_TIME = 10
 
-def get_l2_rx_drop(host, itfs):
-    """
-    Return L2 rx packet drop counter for given interface
-    """
-    res = {}
-    stdout = host.shell("portstat -j")['stdout']
-    match = re.search("Last cached time was.*\n", stdout)
-    if match:
-        stdout = re.sub("Last cached time was.*\n", "", stdout)
-    data = json.loads(stdout)
-    return int(data[itfs]['RX_DRP'])
-
 
 def clear_portstat(dut):
```

Comment:
I understand this function was there before your change.  I just don't see any call to it so perhaps can be removed as part of the code cleanup with this PR?

Reply:
You are right, this function was lying unused. Removed it now in latest commit. Thanks!